pyscf/pbc/df/fft_jk.py

```diff
@@ -41,7 +41,7 @@ def get_j_kpts(mydf, dm_kpts, hermi=1, kpts=np.zeros((1,3)), kpts_band=None):
         kpts : (nkpts, 3) ndarray
 
     Kwargs:
-        kpts_band : (3,) ndarray or (*,3) ndarray
+        kpts_band : ``(3,)`` ndarray or ``(*,3)`` ndarray
```

Comment:
what's the issue of `(3,)` in doc?

Reply:
The issue is that `*` has a meaning in the rst language and should not be used like that. I chose to change both things on the same line.